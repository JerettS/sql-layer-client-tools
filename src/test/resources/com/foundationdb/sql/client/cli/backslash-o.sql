SELECT 1;
\o
SELECT 2;
\o -
SELECT 3;
