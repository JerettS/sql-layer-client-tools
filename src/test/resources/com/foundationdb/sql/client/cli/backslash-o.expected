_SQL_COL_1
------------
          1
(1 row)

 _SQL_COL_1
------------
          2
(1 row)

 _SQL_COL_1
------------
          3
(1 row)

